apsuite/commisslib/measure_bbb_data.py

```diff
@@ -255,14 +255,14 @@ def calc_instant_frequency(data, dtime):
 class BbBLParams(_ParamsBaseClass):
     """."""
 
-    DAC_NBITS = 14
-    SAT_THRES = 2**(DAC_NBITS-1) - 1
-    CALIBRATION_FACTOR = 1000  # [Counts/mA/degree]
-    DAMPING_RATE = 1/13.0  # [Hz]
-    FREQ_RF = 499666000
-    HARM_NUM = 864
-    FREQ_REV = FREQ_RF / HARM_NUM
-    PER_REV = 1 / FREQ_REV
+    DAC_NR_BITS = _asparams.BBBL_DAC_NR_BITS
+    SAT_THRES = _asparams.BBBL_SAT_THRES
+    CALIBRATION_FACTOR = _asparams.BBBL_CALIBRATION_FACTOR
+    DAMPING_RATE = _asparams.BBBL_DAMPING_RATE
+    RF_FREQ = _asparams.RF_FREQ
+    HARM_NR = _asparams.SI_HARM_NR
+    REV_FREQ = _asparams.SI_REV_FREQ
+    PER_REV = 1 / REV_FREQ
```

Comment:
could you please change `PER_REV` to `REV_PER`, @xresende?

Reply:
done